ios/Video/RCTVideo.m

```diff
@@ -2020,7 +2020,6 @@ - (void)pictureInPictureController:(AVPictureInPictureController *)pictureInPict
 }
 
 - (void)pictureInPictureController:(AVPictureInPictureController *)pictureInPictureController restoreUserInterfaceForPictureInPictureStopWithCompletionHandler:(void (^)(BOOL))completionHandler {
-  NSAssert(_restoreUserInterfaceForPIPStopCompletionHandler == NULL, @"restoreUserInterfaceForPIPStopCompletionHandler was not called after picture in picture was exited.");
```

Comment:
@aalzanki You wrote this code 5 years ago, do you remember why you had this assertion? Its crashing our application if we leave it in like this otherwise the app works fine.

Reply:
we fixed it the same way in our variant, but i suppose we should call `playerRef.current?.restoreUserInterfaceForPictureInPictureStopCompleted(true)` after toggling off PiP, according to Apple's documentation 
https://developer.apple.com/documentation/avkit/avpictureinpicturecontrollerdelegate/1614703-pictureinpicturecontroller

> 
> completionHandler
> You must call the completion handler with a value of true to allow the system to finish restoring your player user interface.